sesheta/common.py

```diff
@@ -32,7 +32,7 @@
 _LOGGER = daiquiri.getLogger(__name__)
 
 
-CICD_CONTEXT_ID = "continuous-integration/jenkins/pr-merge"
+CICD_CONTEXT_ID = "local/check"
```

Comment:
should this change to "local/gate" from "local/check" !?

Reply:
I thought thoth-zuul would merge the PR if approved label is added without waiting for reviewer approval, but it actually waits for reviews review.
"local/check" fits best to add labels based upon its results.